modules/persistent-storage-snapshots-creating-deleting.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-snapshots.adoc
+
+[id="persistent-storage-snapshots-using_{context}"]
+= Creating and deleting snapshots
+
+Similar to how a persistent volume claim (PVC) binds to a persistent volume (PV) to provision a volume, `VolumeSnapshotData` and `VolumeSnapshot` are used to create a volume snapshot.
+// k8s docs refer to `VolumeSnapshotContent` instead of `VolumeSnapshotData`. Need to confirm which is correct here.
+
+Volume snapshots must use a supported `PersistentVolume` type.
+
+== Create snapshot
```

Comment:
I would make this procedure a separate module. As a general rule, each module should contain conceptual information, reference information, or a single procedure.